carbon-design-system/carbon
# docs: change Tabs stories for activation prop

changes as per the explanation given in the issue #17005

No breaking changes

Fix the confusing explanation for activation prop for TabsList component and fix the misspelled activation prop in corresponding stories

#### Changelog

**Changed**

- Tabs.mdx change the explanation to `activation` prop
- Tabs.stories.js fix the `automatic` argument to `activation` in Playground story


#### Testing / Reviewing
**Test 1:**

1. Run the `yarn storybook` in `packages/react` project.
2. Go to overview page of Tabs component
3. Go to the section `Tab - tab activation modes`
4. Verify the explanation

**Test 2:**

1. Go to Playground page of Tabs component in storybook
2. Set the `activation` prop to `automatic`
3. Focus a tab in the render view.
4. Use Left or Right keys on keyboard to change the tab selection.
5. Verify content in the tabs panel changes correspondingly.
6. Set the `activation` prop to `manual`
7. Focus a tab in the render view. 
8. Use Left or Right keys on keyboard to change the tab selection.
9. Verify content in the tabs panel does not changes.
10. Select `ENTER` button on keyboard when the focus is on any tab.
11. Verify content in the tabs panel changes correspondingly.
